src/Preview.tsx

```diff
@@ -176,12 +177,30 @@ const Preview: React.FC<PreviewProps> = props => {
     }
   };
 
-  React.useEffect(() => {
+  const onWheelMove: React.WheelEventHandler<HTMLBodyElement> = event => {
+    if (!visible) return;
+    event.preventDefault();
+    const wheelDirection = event.deltaY;
+    setLastWheelZoomDirection(wheelDirection);
```

Comment:
I try in windows, `wheelDirection` always`100` or `-100`, so zoom trigger once, we maybe need change this

Reply:
Very good point, thanks